### DISCUSSION TITLE
Implementation of Quick Monster and NPC creation

### DISCUSSION THREAD

**Discussion by ian612** [wildj79/foundryvtt-starfinder#696] [Q&A] [answered]
Is there any implementation in the system of the "Creating Monsters and Other NPCs" section from the Alien Archive book (Appendix 1)?

If not, is there a module that does this? It seems like something that would be really useful but I wasn't able to locate it in the system.

**Comment by wildj79** [comment, marked as answer]
There is not an implementation within the system.
